Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM healthcatalyst/fabric.baseos:latest
 
 LABEL maintainer="Health Catalyst"
-LABEL version="1.0"
+LABEL version="2.0"
```

Comment:
did you manually change this version?

Reply:
Yes. Since this change represents a change in expectation within the JSON sent to the queue exchange it is wise to bump the major version to indicate it is not backwards compatible with 1.0.